style.css

```diff
@@ -1,5 +1,11 @@
 /* Typography */
 
+
+body {
+  font-size: 100%;
+  line-height: 1.5;
+}
+
 /*
```

Comment:
Can we remove the commented out sections? That just reduces visual noise. We can always use git to iterate and store in progress versions in separate branches.